Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -472,6 +485,7 @@ public Task<object> InvokeCommandAsync(LSP.ExecuteCommandParams request, Cancell
                                 }
                                 capabilities["workspace"]["workspaceFolders"] = true;
                                 capabilities["workspace"]["didChangeWatchedFiles"]["dynamicRegistration"] = true;
+                                capabilities["workspace"]["configuration"] = true;
```

Comment:
Right here @jakebailey

Reply:
That's where the setting is turned on.

Reply:
Thanks, clearly I had the wrong search terms.